cmd/snaptel/flags.go

```diff
@@ -49,10 +49,11 @@ var (
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
-	flPassword = cli.BoolFlag{
```

Comment:
@IzabellaRaulin, using `bool` is a design mistake in my opinion while env var is also supported. The implementation does not work for env var. It's a bug.